code/iaas/auth-logic/src/main/java/io/cattle/platform/iaas/api/auth/github/ProjectResourceManager.java

```diff
@@ -203,19 +142,27 @@ protected Object deleteInternal(String type, String id, final Object obj, ApiReq
         if (!(obj instanceof Account)) {
             return new Object();
```

Comment:
This may be true. I can change that if you like. This was from previous impl and i assumed that there was a reason for it.